egs/wsj/s5/steps/data/reverberate_data_dir.py

```diff
@@ -136,7 +136,10 @@ def pick_item_with_probability(x):
         collection (list or dictionary) where the values contain a field called probability
     """
     if isinstance(x, dict):
-        plist = list(set(x.values()))
+        keylist = list(x.keys())
+        keylist.sort()
+        random.shuffle(keylist)
```

Comment:
Hm.  I don't understand why the order has to be deterministic, but doesn't random.shuffle give you different results each time?  I don't understand how this fixes the issue if nondeterministic order was the problem.

Reply:
Since random seed is set in this script, so you don't need to worry about this.

Reply:
I think for better reproducibility, the order has to be deterministic. Or you will mix an audio with a different noise file every time you run the experiment.